src/components/Timeline/Publication/Actions/Collect/index.tsx

```diff
@@ -0,0 +1,98 @@
+import Loader from '@components/Shared/Loader';
```

Comment:
**question:** can we reuse this from `Publication` folder?

Reply:
Done